Add breaking change from 2.0-2.1 (#22917)
dotnet/docs: includes/core-changes/msbuild/2.1/dotnetclitoolreference.md

### Project tools now included in SDK

> The .NET Core 2.1 SDK now includes common CLI tooling and referencing these from the project is no longer required.

Reviewer: ```suggestion
The .NET Core 2.1 SDK now includes common CLI tooling, and you no longer need to reference these tools from the project.
```

#### Change description

> In .NET Core 2.0, projects reference external .NET tools with the `<DotNetCliToolReference>` project setting. In .NET Core 2.1, some of these tools are now  included with the .NET Core SDK and the setting is no longer needed. If reference to these tools are included in your project, you'll receive an error similar to the following: **The tool 'Microsoft.EntityFrameworkCore.Tools.DotNet' is now included in the .NET Core SDK.**

Reviewer: ```suggestion
In .NET Core 2.0, projects reference external .NET tools with the `<DotNetCliToolReference>` project setting. Starting in .NET Core 2.1, some of these tools are included with the .NET Core SDK, and the setting is no longer needed. If you include references to these tools in your project, you'll receive an error similar to the following: **The tool 'Microsoft.EntityFrameworkCore.Tools.DotNet' is now included in the .NET Core SDK.**
```

Author: `DotNetCliToolReference` was replaced by global/local tools starting in 3.0, considering 2.2 is out of support, I think it's OK to just reference .NET Core 2.1.

How does this sound?

In .NET Core 2.0, projects reference external .NET tools with the `<DotNetCliToolReference>` project setting. In .NET Core 2.1, some of these tools are included with the .NET Core SDK, and the setting is no longer needed. If you include references to these tools in your project, you'll receive an error similar to the following: **The tool 'Microsoft.EntityFrameworkCore.Tools.DotNet' is now included in the .NET Core SDK.**

Reviewer: I like it!
